### PR TITLE
Fixes for Qt 5.12 migration

### PR DESCRIPTION
**Pull request by dobey**

### CODE REVIEW

File: src/qml/MessagesListView.qml [hunk 1]
```diff
@@ -87,7 +87,6 @@ MultipleSelectionListView {
         id: loader
         anchors.left: parent.left
         anchors.right: parent.right
-        height: status == Loader.Ready ? item.height : 0
```
Comment (mariogrip) [inline src/qml/MessagesListView.qml:90]: Why was this here in the first place?

Comment (UniversalSuperBox) [reply src/qml/MessagesListView.qml:90]: Probably to prevent errors when the Loader hadn't completed loading yet: https://github.com/ubports/messaging-app/commit/91f652c8922a572e7e72d9b1e1056582a21bb092

Comment (dobey) [reply src/qml/MessagesListView.qml:90]: No idea. Probably was working around some issue in older Qt, which is now resolved with the improvements to the QtQuick engine.

Comment (lduboeuf) [reply src/qml/MessagesListView.qml:90]: don't see any regressions, i've tested it already before

Comment (UniversalSuperBox) [reply src/qml/MessagesListView.qml:90]: A good test result is a resolution to me.

**Review by UniversalSuperBox — APPROVED** [review]
